cs481-ekh/f21-welolo
# Fixes bug where text box is editable even when an item is selected

Verification steps:
- Select any merchant
- Select any item
- Notice that the payment text box is uneditable with the value of the item selected

- Select any merchant
- Scroll to the bottom of the page and select skip this step
- Notice that the text box is not locked and a value can be entered in
